wiki/Finding-Signatures-with-Ghidra.md

```diff
@@ -0,0 +1,207 @@
+      - **Linux Version**
+         - From Linux: Install the game, make sure proton/compatibility is not enabled, you should see `FTL.x86` and `FTL.amd64` files, this will be 1.6.13 and is the latest version on Steam and is what Hyperspace is built for on Linux.
+         - **From Mac or Windows** you can fetch the linux copy using the [steam console](#opening-the-steam-console) `download_depot 212680 221002` (steam will list where the files were saved)
+      - **Mac Version**
+         - From Mac: Install the game
+         - **From Linux or Windows**: you can fetch the mac copy using the [steam console](#opening-the-steam-console) `download_depot 212680 212682` (steam will list where the files were saved)
+   - **GoG**
+      - **Windows Version**
+         - Download GoG Galaxy
+         - Install FTL
+         - Select an older version of FTL from GoG Galaxy (Todo: add pictures)
+         - For Linux & MacOS you'll have to still use the Windows instructions to get the binary if you want 1.6.9, if you want the latest 1.6.13b (which does have debug symbols also but isn't valid if you're trying to add new hooks for 1.6.9, then you can just download the latest installer from GoG's website)
+      - **Linux Version**
+         - Download the latest Linux installer from the GoG website, it'll be a shell script for `1.6.12` that you'll need to run
+         - **From Windows**: You can probably run this shell script under Windows Subsystem for Linux to get access to the binary it installs
+         - **From Mac**: Unknown but you might be able to run this installer from terminal as-is to get access to the binaries.
+      - **Mac Version**
+         - **From Mac/Windows/Linux**: Download the Offline Installer for 1.6.13, you can open the `.pkg` file directly in Ghidra without installing it.
+   - **Humble Bundle**
+      - **Windows Version**
+         - Not sure, you might need to contact support to get the 1.6.9 installer.
+      - **Linux Version**
+         - **From Linux/Mac/Windows**: Download the `1.6.12` zip directly, unzip somewhere
+      - **Mac Version**
+         - **From Linux/Mac/Windows**: Download the `.dmg` directly, I think Ghidra can open this directly (but untested)
+   - **Epic/Microsoft/Other Sources**
+      No Linux or Mac binaries provided, unknown how to get old Windows binaries other than the downgraders on the forums
+3. Copy all the binaries you wish to examine to their own project folder for Ghidra, you don't need to keep them in their original locations
+4. Open them in Ghidra, see special instructions for each type:
+   - Windows Binaries
+      - After importing the EXE but before opening it you must change the language
+      - ![image](https://user-images.githubusercontent.com/1423894/172201781-e00e2107-4feb-47d3-bd45-2938811b5fcc.png)
```

Comment:
Obviously this works for now because it was manually uploaded in the past, but maybe we should also test if we have a `wiki/img` folder or something and if relative links work?

Reply:
That does in fact work, implemented